src/tools/pub/pubspec.md

```diff
@@ -332,6 +336,22 @@ publish_to: none
 {% endprettify %}
 
 
+### Funding
+
+Package authors may use the `funding` property specify a list of URLs that
+provide information on how users can help fund the development of the package.
+
+{% prettify yaml tag=pre+code %}
```

Comment:
Hmmm. Is dart.dev still using prettify?

Reply:
Only in cases where we need to highlight code, so it's fine to switch this case to a normal markdown code block.